ModernWallet/Classes/Extension/UserDefaults+Extensions.swift

```diff
@@ -14,9 +14,15 @@ extension UserDefaults {
         tempEmail: "tempEmail",
         tempPhone: "tempPhone",
         signUpStep: "SignUpStep",
-        loggedIn: "loggedIn"
+        loggedIn: "loggedIn",
+        menuOrder: "menuOrder"
     )
     
+    var menuOrder: [Int] {
```

Comment:
I think that this variable and also `itemsOrder` in `MenuTableViewController` don't really sound like it is an array of indexes. Probably you can go with `menuIndexes` here and `orderedItems` in `MenuTableViewController` or whatever you can think of. Let me know what you think.